package.json

```diff
@@ -80,5 +80,9 @@
   "engines": {
     "node": ">=14"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "volta": {
+    "node": "18.18.2",
+    "yarn": "1.22.19"
```

Comment:
I've been using node 16. Could we use `master`'s versions? 
```
  "volta": {
    "node": "16.19.1",
    "yarn": "1.22.19"
  }
```

Reply:
Perhaps. Though Node 16 is no longer supported. Works well with Node 18 and loaders.gl is also focused on Node 18. In addition it won't affect your usage unless you are using volta.

Reply:
Ok, I'll make a PR to update deck to 18 where its currently using 16.